contracts/Scheduler/BaseScheduler.sol

```diff
@@ -11,7 +11,32 @@ import "contracts/Library/SchedulerLib.sol";
 contract BaseScheduler is SchedulerInterface {
     using SchedulerLib for SchedulerLib.FutureTransaction;
 
-    address public feeRecipient;       // Recipient of the fee.
+    // The RequestFactory which produces requests for this scheduler.
+    address public factoryAddress;
+
+    // The TemporalUnit (Block or Timestamp) for this scheduler.
+    RequestScheduleLib.TemporalUnit public temporalUnit;
+
+    // The address which will be sent the fee payments.
+    address public feeRecipient;
+
+    // Storage of the transaction parameters.
+    SchedulerLib.FutureTransaction public futureTransaction;
```

Comment:
Can we get rid of these storage variables? We preferably keep them in mem only for a time of transaction.